docs/docsite/rst/dev_guide/developing_collections_creating.rst

```diff
@@ -81,6 +81,15 @@ To learn more about the ``ansible-galaxy`` command-line tool, see the :ref:`ansi
 
 .. _creating_collection_skeletons:
 
+Creating a collection using the ansible-creator tool
+====================================================
+
+The `ansible-creator <https://github.com/ansible/ansible-creator/>`_ tool is designed for effortlessly scaffolding your Ansible content.
+After `installing <https://ansible.readthedocs.io/projects/creator/installing/#installation>`_ ``ansible-creator`` you have two options to interact with it to create a foundational structure for the project:
+
```

Comment:
You might also mention ADT at this point, which is an alternative to installing ansible-creator as a standalone. ADT is a curated package that includes creator so it's worth pointing that out at a high-level.

Reply:
Is it really needed to mention ADT here? The first thing is a recommendation of ADT in the installation guide which I linked.

Reply:
We should decide whether we want the focus on ADT (and VSCode) or (standalone) ansible-creator. Both have detailed information we don't want to repeat here, but if we want the focus on ADT, we can point directly to the ADT docs and skip the ansible-creator docs.

Reply:
I think the focus should be on `ansible-creator` but it's worth pointing out that it is available as part of the ADT bundle.

Reply:
@oraNod @samccann So, do I need to make changes here, or is the PR ready on my side?

Reply:
@xszabo3 I would suggest making the changes recommended here: https://github.com/ansible/ansible-documentation/pull/1302/files#r1583249655